src/Tests/Backend/One/OneToolchain.test.ts

```diff
@@ -31,21 +56,103 @@ suite('Backend', function() {
       });
     });
 
-    // suite('#getToolchainTypes', function() {
+    suite('#getToolchainTypes', function() {
+      test('returns ToolchainCompiler\'s type', function() {
+        const oneCompiler = new OneCompiler();
+        const toolchainTypes = oneCompiler.getToolchainTypes();
+        assert.strictEqual(toolchainTypes.length, 1);
+        assert.strictEqual(toolchainTypes[0], 'latest');
+      });
+    });
 
-    // });
+    suite('#parseVersion', function() {
+      test('returns Version object from string version', function() {
+        const oneCompiler = new OneCompiler();
+        const version1 = oneCompiler.parseVersion('1.0.2~RC0');
+        const version2 = new Version(1, 0, 2, '~RC0');
+        assert.isTrue(version1.equals(version2));
+        assert.isTrue(version2.equals(version1));
+      });
+      test('returns Version object from string version only with major', function() {
+        const oneCompiler = new OneCompiler();
+        const version1 = oneCompiler.parseVersion('1');
+        const version2 = new Version(1, 0, 0);
+        assert.isTrue(version1.equals(version2));
+        assert.isTrue(version2.equals(version1));
+      });
+      test('returns Version object from string version without patch and option', function() {
+        const oneCompiler = new OneCompiler();
+        const version1 = oneCompiler.parseVersion('1.0');
+        const version2 = new Version(1, 0, 0);
+        assert.isTrue(version1.equals(version2));
+        assert.isTrue(version2.equals(version1));
+      });
+      test('returns Version object from string version without option', function() {
+        const oneCompiler = new OneCompiler();
+        const version1 = oneCompiler.parseVersion('1.0.2');
+        const version2 = new Version(1, 0, 2);
+        assert.isTrue(version1.equals(version2));
+        assert.isTrue(version2.equals(version1));
+      });
+      test('returns Version object from string version without minor', function() {
+        const oneCompiler = new OneCompiler();
+        const version1 = oneCompiler.parseVersion('1.0-beta');
+        const version2 = new Version(1, 0, 0, '-beta');
+        assert.isTrue(version1.equals(version2));
+        assert.isTrue(version2.equals(version1));
+      });
+    });
 
-    // suite('#getToolchains', function() {
+    suite('#getToolchains', function() {
```

Comment:
There is no test case of `getToolchains` function. Please added below test cases.

```
      test('get toolchain list', function() {
        // No positive test for get toolchain list
        // To test this test case, prerequisites() function should be run first.
        // However this function needs root permission so it couldn't be executed.
        const oneCompiler = new OneCompiler();
        assert.isDefined(oneCompiler.getToolchains);
      });
```

However, you can add negative test case of `getToolchains()` function.
- request wrong type
- request wrong start and count number

This is the same with `getInstalledToolchains()` function. Please implement as above. :)